camPod/Classes/EditPhoto/VIewModel/PhotoEditorViewModel.swift

```diff
@@ -176,11 +176,11 @@ public class PhotoEditorViewModel {
         let scaleFactorY = CGFloat(cgCropImage!.height)/scaledHeight
 
         if originalCGCropImageWidth > originalCGCropImageHeight {
-            scaledOriginY = (cropView.frame.origin.y - (imageViewHeight - scaledHeight)) * scaleFactorY
+            scaledOriginY = (CGFloat(imageViewHeight - scaledHeight!)) * scaleFactorY
```

Comment:
why force '!' variable value, if you use it in 176 without forcing a value
Naturally forces should be prevented as they could crash your demo and Codacy should give you a warning

Reply:
Codacy does not warn, the variable does not contain the right value when unforced